src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -333,6 +362,10 @@ public static boolean allowsHookUrlOverride() {
             return ALLOW_HOOKURL_OVERRIDE;
         }
 
+        private static ThreadFactory threadFactory() {
```

Comment:
I don't see the value, seems like change for change's sake. The other method is not too complex whereas we'd need to add a three way test to use this as an executor service factory method